### PR TITLE
Liar's Handshake Build

### PR DESCRIPTION
**Pull request by cuzelac**
https://github.com/PhantomsChosen/build-profiles/issues/16

### CODE REVIEW

File: content/builds/hunter/liars-handshake.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+### Options
+* Shotgun
+    * Full Choke makes it much easier to hit all your pellets and activate One-Two Punch
+    * [Grave Robber](https://www.light.gg/db/items/1631667848/grave-robber/) is great because you never have to reload
+    * A lightweight frame gives a convenient mobility boost
+    * One-Two punch is rare on Kinetic shotguns and none currently roll with Grave Robber
+    * The Deicide is a good option in Gambit as it's Void and will deal with the shields on Envoys
+* Mods - these are a good starting point for group activities ([DIM Optimizer Link](https://app.destinyitemmanager.com/optimizer?class=1&p=%7B%22statConstraints%22%3A%5B%7B%22statHash%22%3A392767087%7D%2C%7B%22statHash%22%3A2996146975%7D%2C%7B%22statHash%22%3A1943323491%7D%2C%7B%22statHash%22%3A1735777505%7D%2C%7B%22statHash%22%3A144602215%7D%2C%7B%22statHash%22%3A4244567218%7D%5D%2C%22lockItemEnergyType%22%3Afalse%2C%22upgradeSpendTier%22%3A0%2C%22mods%22%3A%5B3470562293%2C3470562293%2C1484685885%2C3730619869%2C4213142382%2C4044800077%2C1680735357%2C760472183%2C760472183%2C1513970148%2C1513970148%2C3065699930%2C3065699930%2C3836152936%2C2476831315%5D%2C%22exoticArmorHash%22%3A4165919945%7D))
```
Comment (alexlblount) [inline content/builds/hunter/liars-handshake.md]: The front-matter will add a section with this link, so you don't have to add it here.  You can keep it if you like though.

File: content/builds/hunter/liars-handshake.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+description: "PvE build with Liar's Handshake"
+season: 15
+draft: false
+tags: ['hunter', 'pve', "liar's handshake"]
+d2_armor_picker_hash: 'n/a'
+dim_profile_link: 'https://app.destinyitemmanager.com/optimizer?class=1&p=%7B%22statConstraints%22%3A%5B%7B%22statHash%22%3A392767087%7D%2C%7B%22statHash%22%3A2996146975%7D%2C%7B%22statHash%22%3A1943323491%7D%2C%7B%22statHash%22%3A1735777505%7D%2C%7B%22statHash%22%3A144602215%7D%2C%7B%22statHash%22%3A4244567218%7D%5D%2C%22lockItemEnergyType%22%3Afalse%2C%22upgradeSpendTier%22%3A0%2C%22mods%22%3A%5B3470562293%2C1484685885%2C3730619869%2C4213142382%2C4044800077%2C1680735357%2C760472183%2C760472183%2C1513970148%2C1513970148%2C3065699930%2C3065699930%2C3836152936%2C2476831315%5D%2C%22exoticArmorHash%22%3A4165919945%7D'
+
+---
+
+## Overview
```
Comment (alexlblount) [inline content/builds/hunter/liars-handshake.md:11]: Could we add the stats section back from the template? Newer players might find that info useful for adjusting the build to fit their existing armor.

File: content/builds/hunter/liars-handshake.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
+---
+title: "Liars Handshake Hunter-Titan"
+description: "PvE build with Liar's Handshake"
+season: 15
+draft: false
+tags: ['hunter', 'pve', "liar's handshake"]
+d2_armor_picker_hash: 'n/a'
```
Comment (alexlblount) [inline content/builds/hunter/liars-handshake.md]: almost forgot! just leave this as an empty string or remove it.  the value will automatically add a section that will link to d2armorpicker with the hash info, but the hash won't do anything here.

File: content/builds/hunter/liars-handshake.md (hunk 2 above)
Comment (cuzelac) [reply content/builds/hunter/liars-handshake.md:11]: Oops, I saw the template but didn't copy from it when I made this. I'll bring the rest of the formatting up to date to match the template where it doesn't too.

File: content/builds/hunter/liars-handshake.md (hunk 1 above)
Comment (cuzelac) [reply content/builds/hunter/liars-handshake.md]: nah, it's best to keep it in one place; I'll remove this link

**Comment by alexlblount** [comment]
Looks good! Thanks for the build!

**Pull request merged**